lib/signs/utilities/audio.ex

```diff
@@ -323,7 +323,7 @@ defmodule Signs.Utilities.Audio do
 
       # This only occurs at the RJFK mezzanine zone
```

Comment:
Maybe this comment isn't needed anymore? Since the special case is explicit in the pattern (`special_sign: :jfk_mezzanine`).